### DISCUSSION TITLE
how make zed faster (zed working with lags and slow)

### DISCUSSION THREAD

**Discussion by johngoworks** [zed-industries/zed#14236] [Feature Requests]
i use vulkan-swrust (another not work)
[Screencast from 2024-07-11 23-23-26.webm](https://github.com/zed-industries/zed/assets/156248207/77f1c187-d680-4e9d-8593-0477f634e21c)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
